<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryTestDefault</class>
 <widget class="QDialog" name="MemoryTestDefault">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки теста памяти</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../tgca/tgca.qrc">
    <normaloff>:/pictogram/memory.png</normaloff>:/pictogram/memory.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Начальный адрес:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Режим:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Начальные данные:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Конечный адрес:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Приращение адреса:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Приращение данных:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Вывод:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Итераций:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Инверсия:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Устройство:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="mode">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentText">
      <string>r</string>
     </property>
     <item>
      <property name="text">
       <string>r</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>w</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wr</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="startAddress">
     <property name="inputMask">
      <string>HHHHH;_</string>
     </property>
     <property name="text">
      <string>40000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="endAddress">
     <property name="inputMask">
      <string>HHHHH;_</string>
     </property>
     <property name="text">
      <string>BFFFF</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="incAddress">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>200000</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="startData">
     <property name="inputMask">
      <string>HHHHHHHH;_</string>
     </property>
     <property name="text">
      <string>40000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="incData">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="inverse">
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="iteration">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="output">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="device"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../tgca/tgca.qrc"/>
 </resources>
 <connections/>
</ui>
